ui-showdetails/src/main/java/app/tivi/showdetails/details/ShowDetails.kt

```diff
@@ -441,23 +433,46 @@ private fun OverlaidStatusBarAppBar(
     LogCompositions("OverlaidStatusBarAppBar")
 
     Column(modifier) {
-        val props = transition(
-            definition = statusBarTransitionDef,
-            initState = false,
-            toState = showAppBar
+        val transition = updateTransition(showAppBar)
```

Comment:
What about extracting this transition to a function, returning some `data class BarTransition(val offset, val alpha)`?

Reply:
Currently each prop state means that recomposition is next to a minimum, if we extract to group class: 1) We will be wasting a lot of objects. 2) trigger recomposition a lot more.

I think the current code is fine. WDYT?

Reply:
In fact, 2) isn't an issue since they'll be `equals()`

Reply:
I like where this ended up